contracts/contracts/Ramp.sol

```diff
@@ -686,6 +752,22 @@ contract Ramp is Ownable {
         }
     }
 
+    /**
+     * @notice Checks if sustainability fee has been defined, if so sends fee to the fee recipient and intent amount minus fee
+     * to the on-ramper. If sustainability fee is undefined then full intent amount is transferred to on-ramper.
+     */
+    function _transferFunds(bytes32 _intentHash, Intent memory _intent) internal {
```

Comment:
Correct me if I'm wrong, but believe its better practice to emit events in a `public` function rather than inside an internal function. You could calculate `onRampAmount` and `fee` in the internal function, and then perform the transfer and emit event in the `public` function.

Reply:
Seems kinda arbitrary to me? Why would this be a better practice? But given that we don't have multiple on ramp functions any more its not a big deal